repos/spack_repo/builtin/packages/eigen/package.py

```diff
@@ -73,6 +77,7 @@ class Eigen(CMakePackage, ROCmPackage):
 
     depends_on("c", type="build")
     depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
 
```

Comment:
There is a change in the minimum cmake version required. It would be a good time to add this dependency with a version.